include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -307,6 +307,9 @@
 #    define _ONEDPL_CPP20_RANGES_PRESENT 0
 #endif
 
+// Feature public macros
+#define ONEDPL_HAS_RANGES_API  _ONEDPL_CPP20_RANGES_PRESENT
```

Comment:
Let me mention https://github.com/oneapi-src/oneDPL/pull/945, as this is one more feature macro.